QStore4S/src/test/java/edu/asu/qstore4s/controller/QStoreTest.java

```diff
@@ -22,182 +22,172 @@
+        }
+
+    }
+
+    @Test
+    public void testGetData() throws ParserException, IOException, URISyntaxException, ParseException, JSONException,
+            InvalidDataException {
+
+        // Testing for different accept and shallow values
+
+        // accept=application/xml
+        // shallow=true
+
+        {
+            String returnString = qStore.getData(request, response, "application/xml", "true", "test");
+            Assert.assertEquals("inGetXmlShallow", returnString);
+        }
+        // accept=application/json
+        // shallow = true
+        {
+            String returnString = qStore.getData(request, response, "application/json", "true", "test");
+            Assert.assertEquals("inGetJsonShallow", returnString);
+        }
+
+        // accept=application/xml
+        // shallow=false
+        {
+            String returnString = qStore.getData(request, response, "application/xml", "false", "test");
+            Assert.assertEquals("inGetXmlFull", returnString);
+        }
+
+        // accept=application/json
+        // shallow=false
+
+        {
+            String returnString = qStore.getData(request, response, "application/json", "false", "test");
+            Assert.assertEquals("inGetJsonFull", returnString);
+        }
+
+        // accept=application/json
+        // shallow=random
+        // for any shallow random value full method should be called
+
+        {
+            String returnString = qStore.getData(request, response, "application/json", "random", "test");
+            Assert.assertEquals("inGetJsonFull", returnString);
+        }
+
+        // accept=application/json
+        // shallow=null
+        // for shallow null full method should be called
+
+        {
+            String returnString = qStore.getData(request, response, "application/json", null, "test");
+            Assert.assertEquals("inGetJsonFull", returnString);
+        }
+
+    }
+
+    @Test
+    public void testSearch() throws ParserException, IOException, URISyntaxException, ParseException, JSONException,
+            InvalidDataException {
+
+        {
+            String returnString = qStore.search(request, response, "test", "application/xml");
+            Assert.assertEquals("inSearchXml", returnString);
+        }
+
+        {
+            String returnString = qStore.search(request, response, "test", "application/json");
+            Assert.assertEquals("inSearchXml", returnString);
+        }
+
+        {
+            String returnString = qStore.search(request, response, "", "application/json");
+            Assert.assertEquals("{ \"message\" : \"Please provide XML in body of the post request.\" }", returnString);
+        }
+
+        {
+            String returnString = qStore.search(request, response, "  ", "application/xml");
+            Assert.assertEquals("<message>Please provide XML in body of the post request.</message>", returnString);
+        }
+
+    }
+
+    @Test
+    public void testMessage() {

```

Comment:
please follow our guidelines fro test naming. and why is the test for messages in the qstore test class?